src/Backend/Modules/Groups/Actions/Delete.php

```diff
@@ -31,20 +31,19 @@ public function execute(): void
         $this->id = $deleteFormData['id'];
 
         // group exists and id is not null?
```

Comment:
I would remove the comment since it is no longer correct and is just saying what the code said